tests/helpers/plugins.ts

```diff
@@ -48,7 +48,7 @@ export const plugin60: Plugin = {
     is_preinstalled: false,
     created_at: new Date().toISOString(),
     updated_at: new Date().toISOString(),
-    capabilities: {},
+    capabilities: { jobs: [], scheduled_tasks: [], methods: [] }, // inferred on setup
```

Comment:
Maybe we should keep this here, or somewhere, as the `{}` default that is set in postgres? To test that `jobs` and `methods` might not be set on the object.